open-rit.md

```diff
@@ -7,12 +7,9 @@ slug: open-rit
 
 ## What is Open@RIT?
 
-Open@RIT is the new Open Programs Office for the University directed by [Professor Stephen Jacobs](https://www.rit.edu/directory/sxjics-stephen-jacobs).  It's 
-creation was approved by Provost Dr. Ellen Granberg at the end of June 2020 and placed under the Office of the Vice-President of Research, Dr. Ryne Raffaelle 
-at the beginning of August.  
+You can find the Open@RIT Website here: [Open@RIT](https://openr.it/). Open@RIT is the new Open Programs Office for the University directed by [Professor Stephen Jacobs](https://www.rit.edu/directory/sxjics-stephen-jacobs). It’s creation was approved by Provost Dr. Ellen Granberg at the end of June 2020 and placed under the Office of the Vice-President of Research, Dr. Ryne Raffaelle at the beginning of August. 
```

Comment:
i would make this start with "OPEN@RIT now has their own [website](https://openr.it)"

Reply:
Agreed, made the changes.